sdk/go/api.gen.go

```diff
@@ -1972,6 +1958,47 @@ func (r *Host) SetSecretFile(name string, path string) *Secret {
 	}
 }
 
+// HostTunnelOpts contains options for Host.Tunnel
+type HostTunnelOpts struct {
+	// Map each service port to the same port on the host, as if the service were
+	// running natively.
+	//
+	// Note: enabling may result in port conflicts.
+	Native bool
```

Comment:
Does it work like `docker run --network=host`?

Reply:
Yep! Implementation wise I'm sure they're pretty different, but they accomplish the same goal (transparently mapping all container ports to the host).